src/cljs/org/broadinstitute/firecloud_ui/page/method_config_importer.cljs

```diff
@@ -27,6 +29,20 @@
           (style/create-link #(on-back)
             (icons/font-icon {:style {:fontSize "70%" :marginRight "0.5em"}} :angle-left)
             "Back to table")
+          [:div {:style {:marginTop "1em"}}
+           (style/create-section-header (str "Selected Configuration : "
+                                          (mca/get-ordered-name (:config props))))
+           [:div {:style {:width "20%"}}
+            (when (:show-perms-overlay? @state)
+              [mca/AgoraPermsEditor
+               {:is-conf true
+                :selected-entity (:config props)
+                :dismiss-self (fn []
+                                (swap! state assoc :show-perms-overlay? nil))}])
+            [comps/SidebarButton {:width "10%"
+                                  :style :light :margin :top :color :button-blue
+                                  :text "Permissions ..." :icon :gear
```

Comment:
I love this convention (ellipsis to denote opening a dialog box). Would you add a story to make this consistent throughout our buttons?